app/src/main/java/pl/bk20/forest/stats/presentation/StatsDetailsViewModel.kt

```diff
@@ -57,7 +57,7 @@ class StatsDetailsViewModel(
             _day.value = day.value.copy(
                 date = it.date,
                 stepsTaken = it.steps,
-                treeCollected = it.steps >= it.goal,
+                treesCollected = it.steps / it.goal,
```

Comment:
I know it's a simple division, but could you please move it to Day.kt? My head just freezes for a moment to think what this will actually produce. A float or an int? (I know it's integer division and you assign it to an int field, but it would just be more clear.)
Create a getter like `calorieBurned` and name it `treesCollected`/`amountOfTreesCollected` or something like that.